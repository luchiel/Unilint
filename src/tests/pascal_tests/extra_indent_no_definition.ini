[indentation]
indentation_style = spaces
indentation_policy = consistent
extra_indent_for_blocks = consistent
