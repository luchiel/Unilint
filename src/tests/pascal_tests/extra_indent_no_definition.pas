function no_definition(i: integer): boolean;

  function definition(j: integer): boolean;
  begin
    return true;
  end;
