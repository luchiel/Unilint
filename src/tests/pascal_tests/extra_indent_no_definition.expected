3:1: indent too deep
